### PR TITLE
Fixed #1807 and #1797

### LINKED ISSUE #1807: Add `Stylus` to readme
Add `Stylus` to readme under the Languages supported section.
difficulty/newcomer

### PR DESCRIPTION
**Pull request by sgreene570**
Fixed #1807 and #1797

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -94,7 +94,7 @@ To see what coala can do for your language, run:
 +----------------------------+----------------------------+----------------------------+
 | Fortran                    | Python 2                   | YAML                       |
 +----------------------------+----------------------------+----------------------------+
-| Go                         | Python 3                   |                            |
+| Go                         | Python 3                   | Stylus                     |
```
Comment (Makman2) [inline README.rst]: could you try to maintain alphabetical order? I know we have chosen a quite unsuited layout though 😅

**Comment by Makman2** [comment]
@sgreene570 please assign yourself to issues before you start working on them, so we have no double work ;)

File: README.rst (hunk 1 above)
Comment (sgreene570) [inline README.rst]: Sorry for the mix up :) Will mix right away

**Comment by sgreene570** [comment]
My mistake for not assigning this issue earlier, but it seams I cannot assign myself 2 newcomer issues.  My bad for trying to take too much low hanging fruit.

**Comment by Makman2** [comment]
Don't worry 2 issues aren't a problem ;)

**Comment by Makman2** [comment]
ack dad3ba9 60abcf4

**Comment by Makman2** [comment]
hm weird CI problems... maybe try to rebase, hopefully it passes after another run^^ if not we file an issue and merging as soon as CI goes green again :+1: :)

**Comment by sgreene570** [comment]
I am not sure why the CI builds keep failing :sweat_smile:.  Let me know if there is anything else I can do to fix.

**Comment by Makman2** [comment]
Filed https://github.com/coala/coala-bears/issues/1811 because of the CI problems. Seems we aren't the only PR here encountering this^^

**Comment by supergr35** [comment]
rebase and tests will pass

**Comment by sgreene570** [comment]
My mistake, I was rebasing incorrectly  :+1: :)

File: README.rst (hunk 1 above)
Comment (Makman2) [inline README.rst]: hm this was the old commit without alphabetical order :/

**Comment by jayvdb** [comment]
CI problems fixed long ago.
Please rebase this PR, and leave a note on gitter asking for this to be merged.

**Comment by Makman2** [comment]
unack 5cd8733

**Comment by Makman2** [comment]
The list is not in alphabetical order any more :/ Somehow the old commit got in again :/

**Comment by Makman2** [comment]
@sgreene570 why did you close? :D

**Pull request closed without merging**